### DISCUSSION TITLE
In some browsers shortened URL doesn't redirect to the page directly but shows some text instead

### DISCUSSION THREAD

**Discussion by fweng322** [YOURLS/YOURLS#3779] [Q&A]
I created a shortened URL redirecting to a Google Form.   I tested it with my Firefox, Google Chrome and Chromium browser on my laptop (running Ubuntu 22.04) is okay.  However, when I opened it with Google chrome on my Android, it becomes a text page showing link info like "Short link powered by yourls and created ... ago"

![Screenshot_20240704_083540_Chrome](https://github.com/YOURLS/YOURLS/assets/1000315/00d57fd3-975b-4f85-8592-c61f695e16dd)

Another user reported that he opened it with Opera, Google on his desktop computer (running Windows) also has the same problem:

![messageImage_1720052578736](https://github.com/YOURLS/YOURLS/assets/1000315/0dfb8661-fc27-46df-b37c-cdd7ba743c52)

I have to hit "close" there to go to the correct Google Form.

Not all the browsers will have this problem and I have no idea what was wrong.  Any help will be appreciated.

**Comment by dgw** [comment]
Looks like a toolbar plugin, possibly [the sample shipped with YOURLS](https://github.com/YOURLS/YOURLS/blob/97f03117d0a0036b9d0af960ed35e9d0e2e42e0d/user/plugins/sample-toolbar/plugin.php) or a derivative.

Useful information for this thread would have been the YOURLS version and a screenshot of your plugins page (`/admin/plugins.php`).
